components/Date.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+
+interface DateProps {
+  yearDataList: {
+    year: string;
+    team: { title: string; members: { name: string; cargo: string }[] }[];
+  }[];
+  selectedYear: string;
+  onYearChange: (year: string) => void;
+}
+
+const Date: React.FC<DateProps> = ({
+  yearDataList,
+  selectedYear,
+  onYearChange,
+}) => {
+  const selectedYearIndex = yearDataList.findIndex(
+    (yearData) => yearData.year === selectedYear,
+  );
+  const displayedYears = yearDataList.slice(
+    Math.max(0, selectedYearIndex - 2),
+    Math.min(selectedYearIndex + 3, yearDataList.length),
+  );
+
+  return (
+    <div className="flex flex-col items-center justify-center gap-8">
+      <div className="text-5xl font-semibold">Equipa</div>
```

Comment:
This div with the text "Equipa" should be direct inside the `team/page.tsx` file, because it does not make part of the `Date` component. Also, make it as a `<h1>`